lib/dartdoc.dart

```diff
@@ -196,7 +196,8 @@ class DartDoc {
       ..sourceFactory = sourceFactory;
 
     if (packageMeta.isSdk) {
-      libraries.addAll(getSdkLibrariesToDocument(sdk, context));
+      libraries
+          .addAll(new Set()..addAll(getSdkLibrariesToDocument(sdk, context)));
```

Comment:
Alternatively, I guess `getSdkLibrariesToDocument()` might enforce uniqueness.  Would there be any benefit to doing it there?